README.md

```diff
@@ -56,6 +56,29 @@ composer require --dev friendsofphp/php-cs-fixer
 For more details and other installation methods, see
 [installation instructions](./doc/installation.rst).
 
+### Run with Docker
+
+You can take a ready built docker image to run ``php-cs-fixer``.
+
+```console
+docker run -v $(pwd):/code ghcr.io/php-cs-fixer/php-cs-fixer:latest fix src
+```
+
+Or integrate as check into Gitlab CI like this:
+
+```yaml
+php-cs-fixer:
+  image: ghcr.io/php-cs-fixer/php-cs-fixer:3.41-php8.2
+  script:
+    php-cs-fixer fix --diff --dry-run --format=txt src
+```
+
+There are different tags for each stability and php version with syntax `<php-cs-fixer-version>-php<php-version>`. For example:
+
+* `3.41.1-php8.1`
+* `3.41-php8.2`
+* `3-php7.4`
```

Comment:
```suggestion

`$FIXER_VERSION` used in example above is an identifier of a release you want to use, which is based on Fixer and PHP versions combined. There are different tags for each Fixer's SemVer level and PHP version with syntax `<php-cs-fixer-version>-php<php-version>`. For example:

* `3.47.0-php7.4`
* `3.47-php8.0`
* `3-php8.3`
```

Let's change all `3.41` to `3.47` because docs should not point to version that won't be available in the registry. @keradus please let's wait with `3.47.0` until we merge this 🙂. It's not critical, though, we can always fix it later.